css/InputHeader.less

```diff
@@ -24,15 +24,15 @@
 .moon-input-header .moon-input.moon-header-title {
 	position: static;
 }
-.moon-input-header .moon-input.moon-header-title::-webkit-input-placeholder {
+.moon-input-header .moon-input.moon-header-title::-webkit-input-placeholder, .moon-input-header .moon-input.moon-header-title::-moz-placeholder {
```

Comment:
On these 4 additions, could you put a newline between the comma and the period? When the lines get long they are harder to read, and not conveniently line up for comparisons. Thanks!